File: Course-Project/app/src/main/java/com/vladislavmyasnikov/courseproject/data/models/Lecture.java
package com.vladislavmyasnikov.courseproject.data.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Lecture {

    @SerializedName("id")
    public int id;

    @SerializedName("title")
    public String title;

    @SerializedName("tasks")
    public List<TaskInfo> tasks;
}

File: Course-Project/app/src/main/java/com/vladislavmyasnikov/courseproject/data/models/Lecture.kt
package com.vladislavmyasnikov.courseproject.data.models

import com.google.gson.annotations.SerializedName

class Lecture {

    @SerializedName("id")
    var id: Int = 0

    @SerializedName("title")
    var title: String? = null

    @SerializedName("tasks")
    var tasks: List<TaskInfo>? = null
}
